Observability Center of Excellence (#10731)
newrelic/docs-website: src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/observability-coe.mdx

import omaCoeKnowledgeBase from 'images/oma_diagram_coe-knowledge-base.png'

This guide focuses on how to create a Center of Excellence around observability best practices. Adopting an Observability Center of Excellence (OCoE) will help you drive standards, speed and scale across your organization, and maximize the return on the investment in observability solutions.

## What is a Center of Excellence? [#coe-definition]

> A center of excellence (CoE) is a team of dedicated individuals from multiple functional areas of an organization. They're task is to lead the way in exploring and adopting tools, techniques, and practices for the organization.

Reviewer: They're task -> Their task

> A center of excellence (CoE) is a team of dedicated individuals from multiple functional areas of an organization. They're task is to lead the way in exploring and adopting tools, techniques, and practices for the organization.

Author: ```suggestion
A center of excellence (CoE) is a team of dedicated individuals from multiple functional areas of an organization. Their task is to lead the way in exploring and adopting tools, techniques, and practices for the organization.
```

import omaCoeKnowledgeBase from 'images/oma_diagram_coe-knowledge-base.png'

## What is a CoE? [#coe-definition]

By definition, a center of excellence is… 

* A team of dedicated individuals, from multiple functional areas of an organization, that lead the way in exploring and adopting tools, techniques and practices.

* The Center of Excellence promotes collaboration, standards, best practice and adoption. It is supported by a community with a rich collaborative knowledge-base reinforced with frequent enablement and learning.

> CoE's exist around a host of disciplines including things like security, applications, cloud services and automation. This guide focuses on how to create a CoE around observability best practices. This will help you drive standards, speed and scale across your organization, and maximize the return on the investment in observability solutions.

Author: ```suggestion
CoEs exist around a host of disciplines including things like security, applications, cloud services and automation. This guide focuses on how to create a CoE around observability best practices. This will help you drive standards, speed and scale across your organization, and maximize the return on the investment in observability solutions.
```

> CoE's exist around a host of disciplines including things like security, applications, cloud services and automation. This guide is focused on how to create a CoE around observability best practices. This will help you drive standards, speed and scale across your organization, and maximize the return on the investment in observability solutions.

Author: ```suggestion
CoE's exist around a host of disciplines including things like security, applications, cloud services and automation. This guide focuses on how to create a CoE around observability best practices. This will help you drive standards, speed and scale across your organization, and maximize the return on the investment in observability solutions.
```

/>

### Stakeholders [#coe-stakeholders]

The most important elements of the OCoE are the stakeholder groups, who are:
* Responsible for the creation, maintenance and support of the OCoE 
* Responsible for selecting the standards and best practices to be included in the OCoE
* The primary consumers and contributors to the OCoE itself

These stakeholders fall into the three categories (these may be known by other names in your organization): the **core team**, **council**, and **guild**.    

<img
  title="OCoe stakeholders"
  alt="OCoe stakeholders"
  src={omaCoeStakeholders}
/>

> S

Reviewer 2: I think there's a little typo here.

Typically this would be a team of 1-3 people with a background in observability and strong familiarity with New Relic. They work 2-3 hours per week on the OCoE alongside their regular role.

#### Stakeholder 2: Council [#observability-council]

The council stakeholder is group of leaders responsible for selecting or producing, ratifying and promoting observability standards across the organization. 

Their responsibilities include:

* Thought leadership and direction
* Defining and agreeing on standards and tooling
* Ensuring implementation quality

The size of this team will vary based upon your organization's size and structure. It will consist of individuals from across the organization that have a vested interest in ensuring observability standards are implemented and maintained across the business. Ideal candidates for this function include those with leadership roles in operations, development and support. This is an advisory role that does not require a significant commitment of time.

#### Stakeholder 3: Guild [#observability-guild]

> The guild stakeholder is a cross-functional group of individuals across the organization who have experience and passion for observability, tooling, and are willing to help others. They are the *heroes* for your OCoE and the main contributors to supporting others and generating content for it.

Reviewer 2: Could we do without using italics for "heroes"?

> The guild stakeholder is a cross-functional group of individuals across the organization who have experience and passion for observability, tooling, and are willing to help others. They are the *heroes* for your OCoE and the main contributors to supporting others and generating content for it.

Author: ```suggestion
The guild stakeholder is a cross-functional group of individuals across the organization who have experience and passion for observability, tooling, and are willing to help others. They are the heroes for your OCoE and the main contributors to supporting others and generating content for it.
```

  * Observability use cases: determine which Observability Maturity Architecture (OMA) use cases you wish to implement as part of your OCoE. Typically most organizations start with [Service Level Management](/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/optimize-slm-guide); however, based upon your own business focus, other use cases may be a priority. 
  </Collapser>
    
  <Collapser
    id="step7"
    title="7. Recruit observability guild members"
  >
    This final stage of the ramp up is the most important, without the guild members the OCoE will not succeed. However guild members need to volunteer, and as such, this process is about showcasing and evangelizing the successes and benefits of the OCoE to draw more stakeholders to it.
  </Collapser>
</CollapserGroup>

### Collaboration channels [#coe-collab-channels]

<img
  title="OCoe channels"
  alt="OCoe channels"
  src={omaCoeChannels}
/>

> Having determined the collaboration tool of choice for you OCoE, we would recommend setting up a [regular clinic](#coe-clinic) and [collaboration channels](#coe-channels).

Author: ```suggestion
Having determined the collaboration tool of choice for you OCoE, we would recommend setting up a regular [clinic](#coe-clinic) and [collaboration channels](#coe-channels).
```

  <Collapser
    id="recurring-meeting"
    title="4. OCoE catch up (scheduled recurring meeting)"
  >
**Agenda** 
* Progress updates
* Session and engagement planning
* Success stories
* Platform updates
* Support review

**Attendees**
* New Relic CSM/SA/TAM
* Customer core team

  </Collapser>
</CollapserGroup>

## Conclusion [#conclusion]

By implementing some or all of the guidance outlined in this document you will have put in place the key components to create collaborative culture around your observability practice. An environment that will foster passion and develop strong expertise around observability, whilst also delivering business benefits aligned to improving:

* Operational efficiency
* Uptime
* Performance and reliability
* Customer experience 
* Innovation and growth.

> The Observability Center of Excellence helps your organization develop and retain the skills and best practices that are essential to developing and operating today's complex modern application architectures. 

Author: ```suggestion
The OCoE helps your organization develop and retain the skills and best practices that are essential to developing and operating today's complex modern application architectures. 
```
